omnibus/config/software/msodbcsql18.rb

```diff
@@ -10,26 +10,39 @@
 
 # Dynamically build source url and sha256 based on the linux platform and architecture
 # The source url and sha256 are taken from the official Microsoft ODBC Driver for SQL Server page:
+source_url_base = "https://packages.microsoft.com"
+
 if debian_target?
+  base_package_name = "#{name}_#{version}"
   if arm_target?
     # Debian ARM64 build
-    package_name = "#{name}_#{version}_arm64.deb"
-    source url: "https://packages.microsoft.com/debian/12/prod/pool/main/m/msodbcsql18/#{package_name}"
-    source sha256: "d9bb2d2e165e9d86f6b5de75b2baa24c9f0f25107471bcf82254d27f5dafff30"
+    package_name = "#{base_package_name}_arm64.deb"
+    source sha256: "37e692b1517f1229042c743d0f2a7191e0dcb956bbc3785a895aaa6dc328467e"
+    source url: "#{source_url_base}/debian/12/prod/pool/main/m/msodbcsql18/#{package_name}"
   else
     # Debian AMD64 build
-    package_name = "#{name}_#{version}_amd64.deb"
-    source url: "https://packages.microsoft.com/debian/12/prod/pool/main/m/msodbcsql18/#{package_name}"
-    source sha256: "ae8eea58236e46c3f4eae05823cf7f0531ac58f12d90bc24245830b847c052ee"
+    package_name = "#{base_package_name}_amd64.deb"
+    source sha256: "f91004ce72fcd92e686154f90e2a80f4f86469e7cb5f42ef79cba79dc6727890"
+    source url: "#{source_url_base}/debian/12/prod/pool/main/m/msodbcsql18/#{package_name}"
   end
 elsif redhat_target?
-  package_name = "#{name}-#{version}.x86_64.rpm"
-  source url: "https://packages.microsoft.com/rhel/8/prod/Packages/m/#{package_name}"
-  source sha256: "ecd8e148138ee72a452a5357e380580e2c19219c5424f4ac9350cbdf3217fad1"
+  base_package_name = "#{name}-#{version}"
+  if arm_target?
+    # RHEL aarch64 build
+    package_name = "#{base_package_name}.aarch64.rpm"
+    source sha256: "ffbdede1d6af1245ce02cb77d3d5d37648b9121900632d3c6de56fe60d436b12"
+    source url: "#{source_url_base}/rhel/8/prod/Packages/m/#{package_name}"
+  else
+    # RHEL x86_64 build
+    package_name = "#{base_package_name}.x86_64.rpm"
+    source sha256: "8914b1e64c37f3740dff96f7e5db57f40f9564979617ad38acf3d38b958b7cb7"
+    source url: "#{source_url_base}/rhel/8/prod/Packages/m/#{package_name}"
```

Comment:
I don'' remember if this was flagged during the initial review, but this is a higher version that the minimally supported RHEL version the agent itself support, and is likely to crash as soon as the driver attempts to load in RHEL 7

The same goes for debian & suse